components/tfx/README.md

```diff
@@ -0,0 +1,17 @@
+## TFX components for Kubefow Pipelines
+
+Disclaimer: The TFX components in this directory are unofficial and are maintained by the KFP team not the TFX team.
+
+If you experience any issues in this components please create a new issue in the Kubeflow Pipelines repo and assign it to Ark-kun.
+
+These components were created to allow the users to use TFX components in their KFP pipelines, to be able to mix KFP and TFX components.
+
+If your pipeline uses only TFX components, please use the official [TFX SDK](https://www.tensorflow.org/tfx/guide).
+
+See the [sample pipeline](_samples/TFX_pipeline.ipynb) which showcases most of the components.
+
+Aspects and limitations
+* These components use the official TFX container image.
+* These components run the executors and component classes of the official TFX components.
+* These components do not execute TFX drivers, so they do not log metadata themselves (the metadata is logged by the Metadata Writer service instead). The properties of artifacts are currently not logged.
```

Comment:
Perhaps add a reference for TFX drivers?

Reply:
I've tried to find some good documentation for TFX drivers. I can add the link later when we find it.

Reply:
I've added the best link I could find. We can change it later.